http/headers/Permissions-Policy.json

```diff
@@ -1134,7 +1134,9 @@
               "chrome_android": "mirror",
               "edge": "mirror",
               "firefox": {
-                "version_added": false
+                "version_added": "118",
+                "alternative_name": "Feature-Policy: publickey-credentials-get",
+                "notes": "Only supported through the <code>allow</code> attribute on <code>&lt;iframe&gt;</code> elements."
```

Comment:
Do we want to say that this isn't supported if it's only supported on another header?

Reply:
No, because this same pattern of having headers in both files has been followed for all the other entries. I think this is a bit of hangover from the fact that for a while we had a Feature Policy and then it got renamed to Permissions-Policy. Its a real shit that Firefox doesn't swap to the new name.